servers/zelicraft/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "zelicraft network",
```

Comment:
id should match the folder name with no spaces

Reply:
The id is already changed.